site/content/contribute/getting-started/branching.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Mattermost tick-tock Branching Strategy"
+date: 2019-06-18T00:00:00-04:00
+subsection: Getting Started
+---
+
+Mattermost uses a tick-tock release strategy where every other release is a "quality release" that only has bug fixes and no new features. 
+
+The following diagram provides an overview of the branching strategy used to accomplish this. release-5.4 is a feature release and release-5.5 is a quality release. Note the "quality release" branch is based on the previous release branch.
+
+![Branching Overview](/contribute/getting-started/branching-overview.png)
+
+
+## Cherry Pick Process - Developer
+
+When your PR is required on a release branch, you will follow the cherry picking process.
+
+1. Make a PR to master like normal.
+1. Add the appropriate milestone and the `CherryPick/Approved` label.
+1. When your PR is approved it will be assigned back to you to perform the merge and any cherry picking necessary.
+1. Merge the PR
+1. Cherry pick the master commit back to the appropriate releases.
```

Comment:
What happens if the release branch isn't cut already? Do we:
1. Merge now and cherry pick later (I would prefer this and I think this is what most people have one in the past)
2. Merge later and add `Do Not Merge` as a label.